core/src/main/java/io/grpc/internal/HttpUtil.java

```diff
@@ -44,6 +44,18 @@
 public final class HttpUtil {
 
   /**
+   * {@link io.grpc.Metadata.Key} for the :authority pseudo header.
+   */
+  public static final Metadata.Key<String> AUTHORITY_KEY =
+          Metadata.Key.of(":authority", Metadata.ASCII_STRING_MARSHALLER);
+
+  /**
+   * {@link io.grpc.Metadata.Key} for the :path pseudo header.
+   */
+  public static final Metadata.Key<String> PATH_KEY =
```

Comment:
@carl-mastrangelo, are you thinking we should keep it, or preferring to delete it?